python_bindings/CMakeLists.txt

```diff
@@ -49,11 +49,40 @@ if (NOT Halide_ENABLE_RTTI OR NOT Halide_ENABLE_EXCEPTIONS)
     message(FATAL_ERROR "Python bindings require RTTI and exceptions to be enabled.")
 endif ()
 
+##
+# A helper for creating tests with correct PYTHONPATH
+##
+
+function(add_python_test)
+    cmake_parse_arguments(ARG "" "FILE;LABEL" "PYTHONPATH;ENVIRONMENT" ${ARGN})
+
+    list(PREPEND ARG_PYTHONPATH "$<TARGET_FILE_DIR:Halide::Python>/..")
```

Comment:
We'll also need some similar logic (in the `pygen` branch) to construct the proper PYTHONPATH for use when driving Python generators (which aren't tests), but we can refactor that after this merges.